posts/2021-12-13-member-spotlight-automata.md

```diff
@@ -0,0 +1,42 @@
+
+Some of this popularity, we think, also comes from a real desire we are seeing from developers who realize blockchain projects and ecosystems are here to stay, and who are starting to take notice of Rust as their programming language of choice. Polkadot, for instance, is built on an open-source framework called Substrate, which is Rust based, so there’s also a natural inclination there. 
+
+## What was Automata's motivation to join the Rust Foundation?
+
+Developers seem to really enjoy the language, but that’s not to say that there hasn’t been a relatively steep learning curve with Rust. With our participation and close alignment with some of the more active and biggest blockchain communities out there, we aim to help support developers on their journey as they start learning more about Rust - things they wished they might have known from the beginning, hurdles as they start being fully productive, finding the right resources to build on their knowledge, and the list goes on.
+
+Most recently we published the first of our [#AskAutomata technical series](https://medium.com/atanetwork/ask-automata-create-custom-rpc-subscription-in-substrate-c8c90bfd2616) and the first article was a how to guide on creating a custom RPC subscription in Substrate, which is built on the Rust programming language. We want to create more tutorials and developer documentation to help the community get the most out of their experience. 
+
+## What do you hope the Rust Foundation will accomplish in the months and years ahead?
+
+The Rust language official slogan is, “A language empowering everyone to build reliable and efficient software,” but we happen to also read that an earlier version of that sentiment was penned down as, “A language empowering everyone, but especially folks who thought that systems programming wasn’t for them.” Rust is a promising technology, and we hope to continue the same collaborative spirit that has served the programming language from its inception and continue to make it accessible to a wider and growing community. We will do well not to shy away from treating the important work of marketing, documentation and team-building as any less important than development work to continue the Foundation’s incredible success so far.
```

Comment:
```suggestion
The Rust language official slogan is, “A language empowering everyone to build reliable and efficient software,” but we happened to also read that an earlier version of that sentiment was penned down as, “A language empowering everyone, but especially folks who thought that systems programming wasn’t for them.” Rust is a promising technology, and we hope to continue the same collaborative spirit that has served the programming language from its inception and to continue to make it accessible to a wider and growing community. We will do well not to shy away from treating the important work of marketing, documentation and team-building as any less important than development work to continue the Foundation’s incredible success so far.
```

Reply:
We probably want to firm up distinctions between the Foundation, project, and Automata in this paragraph.